.github/workflows/rust.yml

```diff
@@ -10,13 +10,46 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  build:
-
+  rustfmt:
+    name: Rustfmt
     runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: nightly
```

Comment:
What's the reasoning behind having fmt on nightly?

Reply:
Nice observation. Only nightly rustfmt support formatting doc comments. https://rust-lang.github.io/rustfmt/?version=master&search=#format_code_in_doc_comments